### PR TITLE
 Fetch and correctly display data fetched from the service to the actual table with all defined properties

### PR DESCRIPTION
**Pull request by TomassSimko**
This pr includes 

- Fetching all data into the table 
   - I will look later how to tidy up the fillTableMethod to have some other place     
   - Year is now filled with name data however these is then later option to bind it to different data source (api)
- Clickable selection on rows to display description(only title for now)
- Gets rid of () that caused SQL exception 
- Documentation for methods
- Delete functionality implemented 
  - when button in the row is clicked it will get the id -> call the service and act up the result 

Screenshot 2023-01-08 at 18 12 30

### CODE REVIEW

**Comment by TomassSimko** [comment]
[(Needs index) Fetch and correctly display data fetched from the service to the actual table with all defined properties](https://trello.com/c/PtuBtcFY/49-needs-index-fetch-and-correctly-display-data-fetched-from-the-service-to-the-actual-table-with-all-defined-properties)

**Review by LordStacker — APPROVED** [review]
nice work

File: src/main/resources/com/movie_collection/views/movies-view.fxml [hunk 1]
```diff
@@ -1,58 +1,145 @@
...
+<?import javafx.scene.text.TextFlow?>
 
 <AnchorPane maxHeight="-Infinity" maxWidth="-Infinity" minHeight="-Infinity" minWidth="-Infinity" prefHeight="530.0" prefWidth="600.0" xmlns="http://javafx.com/javafx/18" xmlns:fx="http://javafx.com/fxml/1" fx:controller="com.movie_collection.gui.controllers.MovieController">
     <children>
         <StackPane prefHeight="530.0" prefWidth="600.0">
             <children>
                 <VBox prefHeight="200.0" prefWidth="100.0">
                     <children>
-                        <TableView fx:id="moviesTable" prefHeight="320.0" prefWidth="565.0">
+                        <TableView fx:id="moviesTable" editable="true" prefHeight="320.0" prefWidth="565.0">
```
Comment (TomassSimko) [inline src/main/resources/com/movie_collection/views/movies-view.fxml:27]: shiu I need to change this not to be editable

**Review by patrikvalentiny — APPROVED** [review]

**Comment by TomassSimko** [comment]
[(Needs index ) Create description box that if u click on selected movie inside the table u will recieve full additional information](https://trello.com/c/Nw1K2Q89/52-needs-index-create-description-box-that-if-u-click-on-selected-movie-inside-the-table-u-will-recieve-full-additional-information)

**Pull request merged**